NuKeeper/Configuration/CommandLineParser.cs

```diff
@@ -35,8 +35,18 @@ private static RepositoryModeSettings ReadSettingsForRepositoryMode(CommandLineA
             var githubToken = settings.GithubToken;
             var githubRepoUri = settings.GithubRepositoryUri;
 
+            Uri githubApiBase;
+            if (settings.GithubApiEndpoint != null)
+            {
+                githubApiBase = settings.GithubApiEndpoint;
```

Comment:
we aren't taking [this line](https://github.com/NuKeeperDotNet/NuKeeper/blob/master/NuKeeper/Configuration/CommandLineParser.cs#L53) into account - we need the raw github endpoint in order to clone it - unless we get smarter and query github using the org and reponame.

I think for now we have to specify both the api endpoint and the repo uri